<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget_5">
      <property name="styleSheet">
       <string notr="true">Background-color: #EFEAE6</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_25">
       <attribute name="title">
        <string>Leaf Information</string>
       </attribute>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>12</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>271</width>
          <height>191</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">Background-color: #EFEAE6</string>
        </property>
        <property name="title">
         <string>Attribute</string>
        </property>
        <widget class="QLabel" name="label_26">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Units:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Min. Range</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>110</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Max. Range</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="textMinRange">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="textMaxRange">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>130</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="textUnits">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Label: </string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textName">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDescription">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>94</y>
          <width>361</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>71</y>
          <width>68</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_26">
       <attribute name="title">
        <string>Viewing Component Utilities</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_27">
       <attribute name="title">
        <string>Viewing Alternative Consequences</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_28">
       <attribute name="title">
        <string>Subjective Scale</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_29">
       <attribute name="title">
        <string>Quantifying Preferences</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_30">
       <attribute name="title">
        <string>Weight Stability Interval</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
